docs/faq/README.md

```diff
@@ -87,6 +64,42 @@ Regardless of storage option used, IPAM controller expects a directory volume mo
 
 In the mount directory, rename or remove a file named `cis_ipam.sqlite3`.
 
+## Troubleshooting
+
+### How to troubleshoot FIC pod logs ?
+
+`kubectl logs deploy/<name-of-ipam-deployment> -n kube-system -f`
```

Comment:
Do we need to suggest run IPAM controller in debug log level and check the logs for investigation ?